specify/specify7
# Add a dialog with the chrono chart image for geologic table

Fixes #3614


### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- open the chrono data entry form 
- [ ] verify the icon for the chrono chart is present in the header 
- [ ] verify that you can open and close the dialog with the chart 
- [ ] verify it is not present for other tables 
- open the geologictimeperiod tree 
- [ ] verify there is the clock icon next to the search bar in the top bar 
- [ ] verify that you can open and close the dialog with the chart 
- [ ] verify it is not present for other trees
- [ ] verify you can zoom and dezoom on the image 
- [ ] verify you can move the image when zoomed
